its/src/test/java/com/sonar/it/csharp/MultiTargetAppTest.java

```diff
@@ -32,20 +33,21 @@
 import static org.assertj.core.api.Assertions.assertThat;
 
 public class MultiTargetAppTest {
-  @Rule
-  public TemporaryFolder temp = TestUtils.createTempFolder();
 
-  @Before
-  public void init() {
-    TestUtils.reset(ORCHESTRATOR);
+  private static BuildResult buildResult;
```

Comment:
Why this change with buildResult?

Reply:
This is the same with the `BeforeClass` ones. I thought we wanted every `analyzeProject` to be in BeforeClass.
I reverted the changes on these classes.